ReactSkia/textlayoutmanager/RSkTextLayoutManager.cpp

```diff
@@ -72,6 +80,27 @@ int convertFontWeight (FontWeight fontWeight) {
     }
 } 
 
+TextShadow convertShadow (Size offset, double shadowRadius, SkColor shadowColor ){
```

Comment:
In this function; we are not doing any conversion of RN prop value to Skia prop value. 
Assignment of values probably can be done using "TextShadow" constructor itself ; instead of using a separate API for assignment

Reply:
Updated